examples/hello-world/hello.py

```diff
@@ -21,16 +23,22 @@ async def main():
         image_hash="d646d7b93083d817846c2ae5c62c72ca0507782385a2e29291a3d376",
     )
 
-    tasks = [Task(data=None)]
+    tasks = [Task(data=None) for _ in range(7)]
 
-    async with Golem(budget=1.0, subnet_tag="devnet-beta.2") as golem:
-        async for completed in golem.execute_tasks(worker, tasks, payload=package):
-            print(completed.result.stdout)
+    async for completed in golem.execute_tasks(worker, tasks, payload=package, max_workers=3):
+        print(completed.result.stdout)
 
 
 if __name__ == "__main__":
     enable_default_logger(log_file="hello.log")
 
     loop = asyncio.get_event_loop()
     task = loop.create_task(main())
-    loop.run_until_complete(task)
+    try:
+        loop.run_until_complete(task)
+    except KeyboardInterrupt:
+        task.cancel()
+        try:
+            loop.run_until_complete(task)
+        except (asyncio.CancelledError, KeyboardInterrupt):
+            pass
```

Comment:
`hello.py` was supposed to be the simplest example, so I wouldn't handle keyboard interrupt exceptions here (AFAIR other examples include this code).

Reply:
From the PR description:
> modified hello world example, as a **demonstration script** (**changes to be dumped later**)